.github/dependabot.yml

```diff
@@ -18,3 +18,9 @@ updates:
     package-ecosystem: docker
     schedule:
       interval: weekly
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /extra/requirements.txt
```

Comment:
Is this correct? It is not a directory...

Reply:
Wow, quick! :p
Fixed